src/llmcompressor/transformers/sparsification/sparse_model.py

```diff
@@ -20,7 +20,7 @@ def from_pretrained(*args, **kwargs):
         return AutoModelForCausalLM.from_pretrained(*args, **kwargs)
 
 
-def get_shared_processor_src(student: Module, teacher: Optional[Module]) -> str:
+def get_processor_from_model(student: Module, teacher: Optional[Module]) -> str:
```

Comment:
This function doesn't actually get the processor (rtype is `str`, not `Processor`), it gets the processor source. I disagree with this name change.